CDCgov/prime-reportstream
# Handle valueCodeableConcept and valueDateTime for Observation.value in the SR sender transform

This PR correctly handles simple report sending either a valueDateTime or a valueCodeableConcept in `Observation.value`

Test Steps:
1. Using the new FHIR sample test, send as sender configured with the SR transform 
2. Verify no errors occur during conversion

There is also a new automated test

## Changes
- SR transform for Observation.valueDateTime
- Fix the SR transform for Obersvation.valueCodeableConcept
- Add a new integration test that has an Observation with valueDateTime

## Checklist

### Testing

### Process

## Linked Issues

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
